client/src/components/Input.js

```diff
@@ -1,21 +1,14 @@
-import React from 'react';
 import styled from '@emotion/styled';
 
-const InputField = styled.input`
+const Input = styled.input`
+  background: ${props => props.theme.colors.background};
   border: none;
   border-radius: 5px 0px 5px 0px;
```

Comment:
You don't need `border-radius` when the `border: none` is.

Reply:
;) to shape I do.